lang/scala3/Portfile

```diff
@@ -1,59 +1,10 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           select 1.0
-PortGroup           github 1.0
+PortGroup           obsolete 1.0
 
 name                scala3
-categories          lang java
-license             Apache-2
-maintainers         nomaintainer
-description         The Scala Programming Language
-long_description    Scala is a modern multi-paradigm programming language \
-                    designed to express common programming patterns in a \
-                    concise, elegant, and type-safe way.  It smoothly \
-                    integrates features of object-oriented and functional \
-                    languages.  It runs inside a Java Virtual Machine and \
-                    is fully interoperable with Java.
-
-homepage            https://www.scala-lang.org/
-platforms           darwin
-supported_archs     noarch
-
-
-github.setup        lampepfl dotty 3.0.2
-github.tarball_from releases
-distname            ${name}-${version}
-
+replaced_by         scala3.0
+version             3.0.2
 revision            0
```

Comment:
You want to increase the `revision` in this case to make sure users will be "updated" to the `scala3.0` port

Reply:
also `obsolete` ports are retained for a year (i.e., we assume that users at least once a year update their ports...), so it would be useful to add a comment along the lines of `# remove after September 4, 2022`.

Reply:
@reneeotten thanks!